src/middleware/cache/index.ts

```diff
@@ -61,6 +77,14 @@ export const cache = (options: {
     )
   }
 
+  const cacheableStatusCodes = (options.cacheableStatusCodes ?? defaultCacheableStatusCodes).reduce(
```

Comment:
As shown below, it is better to create a new Set() directly.

```suggestion
  const cacheableStatusCodes = new Set(options.cacheableStatusCodes ?? defaultCacheableStatusCodes)
```